src/os/ui/ogc/ogcserver.js

```diff
@@ -857,17 +910,19 @@ os.ui.ogc.OGCServer.prototype.parseWfsCapabilities = function(response, uri) {
     if (op) {
       var getFeatureEl = op.querySelector('Post');
       if (getFeatureEl != null) {
-        if (getFeatureEl.hasAttributeNS(ol.format.XLink.NAMESPACE_URI, 'href')) {
-          this.setWfsUrl(getFeatureEl.getAttributeNS(ol.format.XLink.NAMESPACE_URI, 'href'));
-        } else {
-          var attr = getFeatureEl.attributes[0];
-          // Attr.value is the DOM4 property, while Attr.nodeValue inherited from Node should work on older browsers
-          this.setWfsUrl(attr.value || attr.nodeValue);
+        if (this.parseOperationURLs_) {
+          if (getFeatureEl.hasAttributeNS(ol.format.XLink.NAMESPACE_URI, 'href')) {
+            this.setWfsUrl(getFeatureEl.getAttributeNS(ol.format.XLink.NAMESPACE_URI, 'href'));
+          } else {
+            var attr = getFeatureEl.attributes[0];
+            // Attr.value is the DOM4 property, while Attr.nodeValue inherited from Node should work on older browsers
+            this.setWfsUrl(attr.value || attr.nodeValue);
+          }
         }
         this.setWfsPost(true);
       } else {
         getFeatureEl = op.querySelector('Get');
-        if (getFeatureEl != null) {
+        if (getFeatureEl != null && this.parseOperationURLs_) {
           if (getFeatureEl.hasAttributeNS(ol.format.XLink.NAMESPACE_URI, 'href')) {
             this.setWfsUrl(getFeatureEl.getAttributeNS(ol.format.XLink.NAMESPACE_URI, 'href'));
```

Comment:
Here as well - `setWFSOperationURL`?

Reply:
I ended up removing the separate "operation URL" field and just set the `this.wfsUrl_` and `this.wmsUrl_` directly as it did before. I think the separate operation URL is a step towards possibly keeping track of the different URLs for the different operations in the Capabilities documents, but as just a single URL it seems redundant, since the`this.wmsUrl_` and `this.wfsUrl_` fields are already essentially used as the operation URL in the layer descriptors.